File: docs/apis/reference/snippets/python/auth/account_login.py
import requests
import json

url = "https://openapi.surreal-ai.com/v1/auth/account.login"

payload = json.dumps({
  "username": "cooluser",
  "password": "YipohBe7quoohuev"
})
headers = {
  'Content-Type': 'application/json'
}

response = requests.request("POST", url, headers=headers, data=payload)

print(response.text)

File: docs/apis/reference/snippets/golang/auth/account_login.go
package main

import (
	"fmt"
	"io/ioutil"
	"net/http"
	"strings"
)

func main() {

	url := "https://openapi.surreal-ai.com/v1/auth/account.login"
	method := "POST"

	payload := strings.NewReader(`{
    "username": "cooluser",
    "password": "YipohBe7quoohuev"
}`)

	client := &http.Client{}
	req, err := http.NewRequest(method, url, payload)

	if err != nil {
		fmt.Println(err)
		return
	}
	req.Header.Add("Content-Type", "application/json")

	res, err := client.Do(req)
	if err != nil {
		fmt.Println(err)
		return
	}
	defer res.Body.Close()

	body, err := ioutil.ReadAll(res.Body)
	if err != nil {
		fmt.Println(err)
		return
	}
	fmt.Println(string(body))
}
